server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java

```diff
@@ -111,7 +110,7 @@ public static void completeDefaults(Props props) {
 
   public static Properties defaults() {
     Properties defaults = new Properties();
-    defaults.put(SEARCH_CLUSTER_NAME, "sonarqube");
+//    defaults.put(CLUSTER_NAME, "sonarqube");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)